bftengine/src/bftengine/SimpleClient.cpp

```diff
@@ -170,7 +170,11 @@ namespace bftEngine
 			clientSendsRequestToAllReplicasPeriodThresh{p.clientSendsRequestToAllReplicasPeriodThresh},
 			clientPeriodicResetThresh{p.clientPeriodicResetThresh}
 		{
-				Assert(_fVal >= 1);
+#ifdef ALLOW_0_FAULT_TOLERANCE
+                Assert(_fVal >= 0);
+#else
+                Assert(_fVal >= 1);
```

Comment:
Is it your intention to change the indentation level of line 173 (old)/176 (new)? I am aware this easy to do unintentionally, as I notice this uses tab characters rather than spaces for indents (which is, IMHO, something we should change, but I perceive that revision as unrelated to your current proposed changes, so I would suggest complying with existing formatting until we make such a broader revision).

Reply:
yes. had clang-format disabled on my ide. will update. thanks!